src/libs/Navigation/OnyxTabNavigator.tsx

```diff
@@ -34,6 +46,7 @@ function OnyxTabNavigator({id, selectedTab = '', children, screenListeners, ...r
             {...rest}
             id={id}
             initialRouteName={selectedTab}
+            screenOptions={{animationEnabled: TabNavigatorAnimationEnabled()}}
```

Comment:
Can't we just take this out and avoid the inline object? Something like this:

```suggestion
            screenOptions={screenOptions}
```

Where we define `screenOptions` outside of the component like so: `const screenOptions = {animationEnabled: TabNavigatorAnimationEnabled()}`

Reply:
Of course! Done ✅